src/orion/core/cli/info.py

```diff
@@ -0,0 +1,508 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.core.cli.info` -- Module to info experiments
+========================================================
+
+.. module:: info
+   :platform: Unix
+   :synopsis: List experiments in termnial
+
+"""
+import logging
+
+from orion.core.io.evc_builder import EVCBuilder
+
+log = logging.getLogger(__name__)
+
+
+def add_subparser(parser):
+    """Add the subparser that needs to be used for this command"""
+    info_parser = parser.add_parser('info', help='info help')
+
+    info_parser.add_argument('name')
+
+    info_parser.set_defaults(func=main)
+
+    return info_parser
+
+
+def main(args):
+    """Fetch config and info experiments"""
+    experiment_view = EVCBuilder().build_view_from(args)
+    print(format_info(experiment_view))
+
+
+INFO_TEMPLATE = """\
+{commandline}
+
+{configuration}
+
+{algorithm}
+
+{space}
+
+{metadata}
+
+{refers}
+
+{stats}
+"""
+
+
+def format_info(experiment, templates=None):
```

Comment:
Is the `templates` keyword argument for future configuration?

Reply:
It's to support different formatting, but to be honest it is probably a terrible overkill...